### PR TITLE
Installer Url 404: Cybelsoft.DriversCloud v11.1.0.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/54788)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
```
--> Installing the Manifest 5.1.1

Found IO Ninja [tibbo.ioninja] Version 5.1.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-amd64.msi
  ██████████████████████████████  23.7 MB / 23.7 MB
Installer hash does not match; this cannot be overridden when running as admin
```

File: manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2022-03-15
+Installers:
+- Architecture: x86
+  InstallerUrl: http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-x86.msi
```
Comment (ItzLevvie) [inline manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml:17]:
```suggestion
  InstallerUrl: http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-x86-core.msi
```

File: manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  InstallerSha256: B11EA4B62B7CB1578C3EC86EF6DC14366C6C81DC47E8CD4DA70191EA413BDFAA
+  ProductCode: '{8365054C-A397-4F89-9FEC-76F9764A1F7E}'
+- Architecture: x64
+  InstallerUrl: http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-amd64.msi
```
Comment (ItzLevvie) [inline manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml:21]:
```suggestion
  InstallerUrl: http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-amd64-core.msi
```

File: manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2022-03-15
+Installers:
+- Architecture: x86
+  InstallerUrl: http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-x86.msi
+  InstallerSha256: B11EA4B62B7CB1578C3EC86EF6DC14366C6C81DC47E8CD4DA70191EA413BDFAA
+  ProductCode: '{8365054C-A397-4F89-9FEC-76F9764A1F7E}'
```
Comment (ItzLevvie) [inline manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml:19]:
```suggestion
  ProductCode: '{848589EE-D3DD-43EA-A9E8-FBE138BB5504}'
```

File: manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2022-03-15
+Installers:
+- Architecture: x86
+  InstallerUrl: http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-x86.msi
+  InstallerSha256: B11EA4B62B7CB1578C3EC86EF6DC14366C6C81DC47E8CD4DA70191EA413BDFAA
```
Comment (ItzLevvie) [inline manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml:18]:
```suggestion
  InstallerSha256: 9069FA67B6971FB66677D322B81B9D4C7D3AAC62F392AEA4AFE3D236FA751F2D
```

File: manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  ProductCode: '{8365054C-A397-4F89-9FEC-76F9764A1F7E}'
+- Architecture: x64
+  InstallerUrl: http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-amd64.msi
+  InstallerSha256: 4A5EFCD206C3C1B890C8CC5F695AD57A0DE65E31F1447B64D083A629BD0C377A
+  ProductCode: '{FB6C9D37-258C-4FE0-8564-C87FF62897D9}'
```
Comment (ItzLevvie) [inline manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml:23]:
```suggestion
  ProductCode: '{746A9073-B219-4128-827D-D93AFA93F9DE}'
```

File: manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+UpgradeBehavior: install
+ReleaseDate: 2022-03-15
+Installers:
+- Architecture: x86
+  InstallerUrl: http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-x86.msi
+  InstallerSha256: B11EA4B62B7CB1578C3EC86EF6DC14366C6C81DC47E8CD4DA70191EA413BDFAA
+  ProductCode: '{8365054C-A397-4F89-9FEC-76F9764A1F7E}'
+- Architecture: x64
+  InstallerUrl: http://tibbo.com/downloads/archive/ioninja/ioninja-5.1.1/ioninja-5.1.1-windows-amd64.msi
+  InstallerSha256: 4A5EFCD206C3C1B890C8CC5F695AD57A0DE65E31F1447B64D083A629BD0C377A
```
Comment (ItzLevvie) [inline manifests/t/tibbo/ioninja/5.1.1/tibbo.ioninja.installer.yaml:22]:
```suggestion
  InstallerSha256: 6A548F80079CC349A94C5D7092CBAE7A731F21C3349CDEB1B9EF2631ACF53972
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Hash mismatch is a Delivery Optimization issue; works fine on WinINet but the installers here are meant to be the core version instead of the version that requires driver installation.

**Review by ItzLevvie — DISMISSED** [review]

**Comment by vedantmgoyal2009** [comment]
Hello, dumb bot, don't you know it is a draft pull request.

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**